install/kubernetes/helm/istio/values.yaml

```diff
@@ -446,3 +446,39 @@ kiali:
   dashboard:
     username: admin
     password: admin
+
+certmanager:
+  enabled: false
+  replicaCount: 1
+  image:
+    repository: quay.io/jetstack/cert-manager-controller
+    tag: v0.3.1
+    pullPolicy: IfNotPresent
```

Comment:
Use the global pull policy, I think all others are converted.

Reply:
done